### PR TITLE
(DOCSP-31516) Changes listed values for --events to refer to API docs

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

A user raised the issue that the listed event types for the --event description was not complete. We should point them to the OpenAPI spec for a more complete list of possible values that stays up to date with changes. I also adjusted the code so that commands for OM and CM point to those respective docs (dochub links created for all 3)

_Jira ticket:_ 

https://jira.mongodb.org/browse/DOCSP-31516

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by gssbzn — DISMISSED** [review]
LGTM

**Comment by sarahsimpers** [comment]
@gssbzn It looks like this touched some MCLI CM/OM commands... would the values in the OpenAPI spec apply to them as well, or should I create a separate description for those to leave as-is?

**Review by gssbzn — COMMENTED** [review]
A new description would probably be best linking to https://www.mongodb.com/docs/cloud-manager/reference/api/alert-configurations-create-config/ and https://www.mongodb.com/docs/cloud-manager/reference/api/alert-configurations-create-config/ (or doc hub equivalent)

**Comment by sarahsimpers** [comment]
> A new description would probably be best linking to https://www.mongodb.com/docs/cloud-manager/reference/api/alert-configurations-create-config/ and https://www.mongodb.com/docs/cloud-manager/reference/api/alert-configurations-create-config/ (or doc hub equivalent)

Done! Ready for another look whenever you have time

File: internal/usage/usage.go [hunk 1]
```diff
@@ -139,7 +139,8 @@ dbName and collection are required only for built-in roles.`
...
-	Event                                     = "Type of event that triggered the alert. Valid values are AWS_ENCRYPTION_KEY_NEEDS_ROTATION, AZURE_ENCRYPTION_KEY_NEEDS_ROTATION, CLUSTER_MONGOS_IS_MISSING, CPS_RESTORE_FAILED, CPS_RESTORE_SUCCESSFUL, CPS_SNAPSHOT_BEHIND, CPS_SNAPSHOT_DOWNLOAD_REQUEST_FAILED, CPS_SNAPSHOT_FALLBACK_FAILED, CPS_SNAPSHOT_FALLBACK_SUCCESSFUL, CPS_SNAPSHOT_SUCCESSFUL, CREDIT_CARD_ABOUT_TO_EXPIRE, DAILY_BILL_OVER_THRESHOLD, GCP_ENCRYPTION_KEY_NEEDS_ROTATION, HOST_DOWN, JOINED_GROUP, NDS_X509_USER_AUTHENTICATION_CUSTOMER_CA_EXPIRATION_CHECK, NDS_X509_USER_AUTHENTICATION_MANAGED_USER_CERTS_EXPIRATION_CHECK, NO_PRIMARY, OUTSIDE_METRIC_THRESHOLD, OUTSIDE_SERVERLESS_METRIC_THRESHOLD, OUTSIDE_REALM_METRIC_THRESHOLD, PENDING_INVOICE_OVER_THRESHOLD, PRIMARY_ELECTED, REMOVED_FROM_GROUP, REPLICATION_OPLOG_WINDOW_RUNNING_OUT, TOO_MANY_ELECTIONS, USER_ROLES_CHANGED_AUDIT, USERS_WITHOUT_MULTIFACTOR_AUTH."
+	Event                                     = "Type of event that triggered the alert. To learn which values the CLI accepts, see the Enum for eventTypeName in the Atlas API spec: https://dochub.mongodb.org/core/atlas-event-names."
```
Comment (jwilliams-mongo) [inline internal/usage/usage.go:142]: we have more than one Atlas API (admin API / data API). I think we can add `Admin` here for clarify.
```suggestion
	Event                                     = "Type of event that triggered the alert. To learn which values the CLI accepts, see the Enum for eventTypeName in the Atlas Admin API spec: https://dochub.mongodb.org/core/atlas-event-names."
```

**Review by jwilliams-mongo — DISMISSED** [review]
@sarahsimpers LGTM with one optional suggestion for clarity

**Review by gssbzn — APPROVED** [review]
💯

**Review by wtrocki — APPROVED** [review]

**Pull request merged**